src/components/dataItem/filter/FilterRow.js

```diff
@@ -18,7 +18,8 @@ const FilterRow = ({
         return null
     }
 
-    const dataItem = dimension && dataItems.filter((d) => d.id === dimension)[0]
+    const dataItem = dataItems.find((d) => d.id === dimension)
```

Comment:
What about combining these into a single line to avoid having another variable with a name similar to other variables?

Reply:
Done!